peachjam/js/components/tabs.ts

```diff
@@ -0,0 +1,21 @@
+export class ToggleTab {
+  btn: HTMLElement;
+
+  constructor (btn: HTMLElement) {
```

Comment:
instead btn I would call it `anchor` since we're getting an `href` attribute from the element. 
If we want to be more explicit, perhaps we could even change HTMLElement to `HTMLAnchorElement`

Reply:
done